rules.md

```diff
@@ -218,11 +189,9 @@ Once a Task Claim is approved by an Admin, the Task Reward will be awarded to th
 
 ## 2.7 Rewards
 
-Incubator is a pure-Dash fund meaning all rewards are awarded in Dash.
-
-Each type of [Task](#25-tasks) offered by the Incubator has limits on the min/max rewards an Admin may [price](#433-pricing-tasks) as defined below.
+The Incubator is a pure Dash fund, meaning all rewards are priced and paid in DASH units.
 
-The Admin % is an added [commission reward](#44-admin-rewards) that an Admin earns for Approving a Claim on a Task they own, plus a [commission bonus](#441-commission-bonuses) based on the Admin's level of activity.
+Admins can price tasks up to the max price.  Strategists can claim a commission up to the max commission listed in the table below.  They can either keep the full claimed [reward](#44-admin-rewards) or split it with a non-Strategist admin who has assisted in the administration process.
```

Comment:
... IMO hopefully the Lead Strategist role (wherever it'd be linked) also has an oversight role along with the strategists, especially when I see "up to the max price". Just making sure that definition is easy to find.

Reply:
Lead Strategist rights/responsibilities are outlined in Section 4.7 and section 4.5 is about governance.  Are you saying you think the LS should have veto power on approving tasks?  I don't think the LS should have that explicit power, but they are able to adjust Strategist budgets (which haven't already been given).  Strategists have full autonomy over the budget they have been given.  

The strategic committee can intervene to remove Stategists who are out of line.  This rule update removes the power to of the LS to unilaterally remove Strategists.

Reply:
I see these references now. No need to add further veto power on task approvals. Thanks.